### DISCUSSION TITLE
How to use acheong08/ChatGPT with shell_gpt

### DISCUSSION THREAD

**Discussion by ashishbinu** [TheR1D/shell_gpt#274] [Ideas]
Is it possible to use [revChatGPT.V1](https://github.com/acheong08/ChatGPT/) with shell_gpt

**Comment by TheR1D** [comment]
No, currently it is not possible. Shell-GPT can be used only with official OpenAI API, or if you will manage to make an API and replicate `/completion` endpoint, you could point `sgpt` to your reverse proxy using `OPENAI_API_HOST` in config file.

**Comment by ashishbinu** [comment]
Let me see. I will give it a shot

**Reply by TheR1D** [reply]
An alternative approach might be to utilize LocalAI in conjunction with the local LLM. The most recent update to ShellGPT (see https://github.com/TheR1D/shell_gpt/discussions/308) has incorporated support for LocalAI.
